options.js

```diff
@@ -2,10 +2,15 @@
   const list = document.getElementById("saved-urls");
   var urls = "";
   chrome.storage.local.get("scroll-mark", function(result) {
+    console.log(result);
     urls = result["scroll-mark"];
     for (var url in urls) {
       var div = document.createElement("div");
-      div.innerHTML = "<a href=" + url + ">" + url + "</a>";
+      let percentage = Math.round((urls[url].offset / urls[url].total) * 100);
+      if (percentage)
+        div.innerHTML =
+          "<a href=" + url + ">" + url + "</a> (" + percentage + "% completed)";
```

Comment:
Maybe "...% scrolled" would be more relatable.

Reply:
Just % will do I believe